CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+### 0.11.0 - Sep 15, 2023
+
+* Add event callback for events like 'slotmap updated'.
+* Add connect callback for the sync API.
+* Add redisClusterAsyncConnect2() in API.
```

Comment:
Maybe mention what it does? (e.g. "... for fully asynchronous startup" or something.) For other bullets, we mention what the features do but not always the function names.

Reply:
Updated